src/shared/codelens/localLambdaRunner.ts

```diff
@@ -653,11 +668,19 @@ async function waitForDebugPort({
         SAM_LOCAL_PORT_CHECK_RETRY_TIMEOUT_MILLIS_DEFAULT
     )
 
-    await tcpPortUsed.waitUntilUsed(
-        debugPort,
-        SAM_LOCAL_PORT_CHECK_RETRY_INTERVAL_MILLIS,
-        timeoutMillis
-    )
+    try {
+        await tcpPortUsed.waitUntilUsed(
+            debugPort,
+            SAM_LOCAL_PORT_CHECK_RETRY_INTERVAL_MILLIS,
+            timeoutMillis
+        )
+
+        return true
+    } catch (err) {
+        channelLogger.logger.verbose(`Failed to wait for port ${debugPort} to open: ${err}`)
```

Comment:
What do you think of something like `timed out after ${timeoutMillis}ms waiting for port ${debugPort} to open: ${err}`